packages/transfer-app/src/SentBalance.tsx

```diff
@@ -0,0 +1,155 @@
+// Copyright 2018-2019 @paritytech/substrate-light-ui authors & contributors
+// This software may be modified and distributed under the terms
+// of the Apache-2.0 license. See the LICENSE file for details.
+
+import { SubmittableResult } from '@polkadot/api/SubmittableExtrinsic';
+import IdentityIcon from '@polkadot/ui-identicon';
+import { logger } from '@polkadot/util';
+import { ApiContext } from '@substrate/ui-api';
+import { Icon, Margin, Stacked, SubHeader } from '@substrate/ui-components';
+import BN from 'bn.js';
+import { Subscription } from 'rxjs';
+import React from 'react';
+import { RouteComponentProps, Redirect } from 'react-router';
+
+import { BlueHeader, InlineSubHeader } from './SentBalance.styles';
+import { MatchParams, TransferParams } from './types';
+
+interface Props extends RouteComponentProps<MatchParams, {}, Partial<TransferParams>> { }
+
+interface State {
+  showDetails: boolean;
+  txResult?: SubmittableResult;
+}
+
+const l = logger('transfer-app');
+
+export class SentBalance extends React.PureComponent<Props, State> {
+  static contextType = ApiContext;
+
+  private subscription: Subscription | undefined;
+
+  context!: React.ContextType<typeof ApiContext>; // http://bit.ly/typescript-and-react-context
+
+  state: State = {
+    showDetails: false
+  };
+
+  componentDidMount () {
+    const { api, keyring } = this.context;
+    const { location: { state }, match: { params: { currentAddress } } } = this.props;
+
+    if (!state || !(state.amount instanceof BN) || !state.recipientAddress) {
+      // This happens when we refresh the page while a tx is sending. In this
+      // case, we just redirect to the send tx page.
+      return;
+    }
+
+    const { amount, recipientAddress } = state;
+    const senderPair = keyring.getPair(currentAddress);
+
+    l.log('sending tx from', currentAddress, 'to', recipientAddress, 'of amount', amount);
+
+    // Send the tx
+    // TODO Use React context to save it if we come back later.
+    // retrieve nonce for the account
+    this.subscription = api.tx.balances
+      // create transfer
+      .transfer(recipientAddress, amount)
+      // send the transaction
+      .signAndSend(senderPair)
+      .subscribe((txResult) => {
```

Comment:
`.subscribe((txResult: SubmittableResult) => {})`

Reply:
Type is inferred here. Still added it